### PR TITLE
GameWisp Provider

### PR DESCRIPTION
**Pull request by hootener**
Added support for GameWisp. A fan funding service for livestreamers. 

This is a resubmission of [this closed PR](https://github.com/SocialiteProviders/Providers/pull/82).

### CODE REVIEW

File: build/split.sh [hunk 1]
```diff
@@ -66,6 +66,7 @@ split Fitbit                src/Fitbit:git@github.com:SocialiteProviders/Fitbit.
 split Flattr                src/Flattr:git@github.com:SocialiteProviders/Flattr.git                         "master"
 split Flickr                src/Flickr:git@github.com:SocialiteProviders/Flickr.git                         "master"
 split Foursquare            src/Foursquare:git@github.com:SocialiteProviders/Foursquare.git                 "master"
+split GameWisp              src/GitLab:git@github.com:SocialiteProviders/GameWisp.git                       "master"
```
Comment (faustbrian) [inline build/split.sh]: @hootener GitLab should be GameWisp

Comment (hootener) [inline build/split.sh]: Apologies. Looks like it's time for more coffee....

**Comment by faustbrian** [comment]
Will move this when I get home.

**Comment by hootener** [comment]
Eed

**Pull request merged**